documentation/puppet-api/v4/catalog_resource.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
This is copied directly from the Puppet repo. Not sure if this is a good idea to duplicate this here, but I wasn't sure what else to do, not being terribly familiar with JSON Schema. Thoughts?

Reply:
I thought there was a way to reference Puppet Docs from within server markdown pages.

Reply:
I think puppet_conf_setting_diffs.markdown contains links like what we could use. Maybe @jbondpdx has opinions on this?